cassdegrees/ui/views/student.py

```diff
@@ -93,7 +95,7 @@ def student_create(request):
     if id_to_view:
         # Create a new cookie in the default plan location containing compressed relevant plan details
         request.session['plan:'] = compress(
-            {'program_id': int(id_to_view), 'date': timezone.localtime().strftime('%d/%m/%Y %H:%M')}
+            {'program_id': int(id_to_view), 'date': timezone.now().strftime('%d/%m/%Y')}
```

Comment:
Is this part of this PR?

Reply:
This is the result of a merge with my branch